Framework/API/inc/MantidAPI/Expression.h

```diff
@@ -116,6 +116,8 @@ class MANTID_API_DLL Expression {
    */
   void toList(const std::string &sep = ",");
 
+  static const std::vector<std::string> DEFAULT_OPS_STR;
```

Comment:
We do a fair amount of allocations with statics as startup. Would it be possible to switch to using an `std::array<std::string_view>` for this, e.g. https://coliru.stacked-crooked.com/a/a86a8130a624ce70?

Reply:
Hello Martyn, sorry we did not reply earlier, we were all on vacation.
I suggest we will discuss that when Dominik is back (September 1st).

Reply:
Previously, there was a global variable holding a list of `char *` at the `Mantid::API` namespace level that was subsequently cast to a `std::vector<std::string>`. I replaced the list of `char *` with a `static const std::vector<std::string>`, so there is no need to cast anymore. If there is any speed up possible by replacing `std::string` with `std::string_view`, then it might be lost as either a new `add_operator` function overload needs to be created that accepts `std::string_view` instead of `std::string`, or a cast needs to be performed. Let me know if I am missing something here and whether you would like to pursue this as proposed by you.